python/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+if(FINUFFT_USE_CPU)
+    install(TARGETS finufft LIBRARY DESTINATION finufft)
```

Comment:
Can you briefly explain what this does? Does it just install the library into some destination that scikit-build then picks up and moves it to the appropriate place (the `python/finufft/ subdirectory)?

Reply:
To my understanding it install the library in the wheel finufft